client/v1/types.go

```diff
@@ -115,7 +115,85 @@ type AKSConfig struct {
 }
 
 type RKEConfig struct {
-	//TBD
+	// Kubernetes nodes
+	Hosts []Host `yaml:"hosts"`
+	// Kubernetes components
+	Services Services `yaml:"services"`
+}
+
+type Host struct {
```

Comment:
Can it be RKEConfigHost/RKEConfigServices. And ETCDService/KubeControllerService/etc